Detectors/CTP/workflow/src/WriterSpec.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
```

Comment:
Also this one better to rename to DigitWriterSpec and to have it in the WorkflowIO

Reply:
I rename. I do not understand second part of sentence: " to have it in the WorkflowIO" ?

Reply:
I mean put reader and writer workflows in a separate library, depending only on the data formats. E.g. this can be WorkflowIO directory creating a library of the same name. Or, see how TRD did this, creating `io` directory in the `O2/Detectors/TRD/workflow`